### ISSUE TITLE
After user sets his password, he should be sent to his profile page in view mode

### ISSUE THREAD

**Issue by treasuretron** [spacebase/spacebasenz#47]
In GitLab by @richbodo on Feb 8, 2018, 19:35

Low priority.

Page:  http://spacebase.co/user/96/edit?pass-reset-token=blah_blah

* Expected:

When clicking on an email verification link, user is sent to a page like the one above.

He then sets his password, and should be redirected to his user profile page, in view mode, not edit mode.

* Observed:

Password is set, but user remains on the initial password set page.

**Comment by treasuretron** [comment]
In GitLab by @aspiringweb on Feb 20, 2018, 12:25

assigned to @aspiringweb

**Comment by treasuretron** [comment]
In GitLab by @aspiringweb on Feb 20, 2018, 12:26

assigned to @StephenCataldo and unassigned @aspiringweb

**Comment by treasuretron** [comment]
In GitLab by @StephenCataldo on Feb 21, 2018, 19:31

mentioned in merge request !65

**Comment by treasuretron** [comment]
In GitLab by @StephenCataldo on Feb 22, 2018, 12:53

@richbodo 
This focuses new members, when they submit their edit-profile = account page to set their password, to go to their profile. My intuition is that this should be the general flow: any time the profile is edited, for passwords or other things, it goes to the profile-view, whether done by new users or old, or by an admin. I set up the redirect this way so far; it would be easy to check for the path to limit this for email verification / forgot password links.

**Comment by treasuretron** [comment]
In GitLab by @StephenCataldo on Feb 23, 2018, 12:31

marked this issue as related to #43

**Comment by treasuretron** [comment]
In GitLab by @richbodo on Feb 23, 2018, 19:21

Assuming this fix is on dev and awaiting qa there.

**Comment by treasuretron** [comment]
In GitLab by @richbodo on Feb 23, 2018, 19:42

Nope.  Not on dev.  Where is this one?

**Comment by treasuretron** [comment]
In GitLab by @richbodo on Mar 1, 2018, 19:26

@StephenCataldo this is correct:

any time the profile is edited, for passwords or other things, it goes to the profile-view, whether done by new users or old, or by an admin. I set up the redirect this way so far; it would be easy to check for the path to limit this for email verification / forgot password links.

And I've tested that as working on dev.
